res/xml/colemak.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard botton_row="false">
+  <row>
+    <key key0="q" key2="1" key4="esc"/>
+    <key key0="w" key1="~" key2="2" key3="\@" key4="!"/>
+    <key key0="f" key1="£" key2="3" key3="\#" key4="€"/>
+    <key key0="p" key2="4" key3="$" key4="f11_placeholder"/>
+    <key key0="g" key2="5" key3="%" key4="f12_placeholder"/>
+    <key key0="j" key1="accent_caron" key2="6" key3="^" key4="accent_trema"/>
+    <key key0="l" key2="7" key3="&amp;"/>
+    <key key0="u" key2="8" key3="*"/>
+    <key key0="y" key2="9" />
+    <key key0="," key1="0" key3="."/>
+
+  </row>
+  <row>
+    <key key0="a" key2="tab"/>
+    <key key0="r" key1="accent_grave" key2="accent_tilde"/>
+    <key key0="s" key1="`" key2="~" />
+    <key key0="t" key1="accent_double_aigu" key2="accent_ring"/>
+    <key key0="d" key1="&quot;" key2="'" />
+    <key key0="h"  />
+    <key key0="n" key1="accent_ogonek"/>
+    <key key0="e" key1="accent_circonflexe" key2="accent_aigu" />
+    <key key0="i" key1="\\" key2="/"/>
+    <key key0="o" />
+
+  </row>
+  <row>
+    <key key0="shift" width="1.5" />
+    <key key0="z" />
+    <key key0="x" key1="&gt;" key2="&lt;"/>
+    <key key0="c" key1="{" key2="}" key3="accent_cedille"/>
+    <key key0="v" key1="[" key2="]"/>
+    <key key0="b" key1="(" key2=")"/>
+    <key key0="k" key1=";" key2=":"/>
+    <key key0="m" key1="|" key2="\?" />
+    <key key0="backspace" shift="0.25" key1="delete" width="1.25"/>
```

Comment:
Why this empty space between `m` and backspace ?

Reply:
This is because if they are together you can press delete instead of m

Reply:
Makes sense but it's the only layout that have this. I wonder if this should be an option available to everyone.
Anyway, I don't mind this.

Reply:
Yes it could be an option, I haven't seen anything related to the app code I`ll try to find out where are the interfaces and try to place an option for that.

Reply:
Let's not worry about the option yet as I'm not yet sure it is useful. Let's keep the gap in the layout and see how it is on the long run. Maybe other people will share their opinions.

Reply:
Ok I`ve take a look on how to put this option but it's another issue.
I've based this layout, the character keys, in the picture bellow:
https://lh5.ggpht.com/Y6MxBScWfyORVaMMkgMAO3Kl63ULQXTisNYIjCZdeU2c9hIy_j4bvXZyWoPms_I02_NA=h500
But I didn't followed the layout for symbols and had tried to put it when I thought that it was better.
I think that this ok, I'll keep using it and see if there are any keys that could be changed, please do a review to see if all necessary keys are here.
# this is another issue
Can the accents be merge in to a one key with their characters? Like merging "accent_tilde" with ~ character, if the user press it and press a compatible key like a it became ã, if the user press it and space it became ~.

Reply:
> Can the accents be merge in to a one key with their characters?

No, this is not possible.

Reply:
This in fact is exact the behavior, but now we have to add like accent_tilde and ~ keys, but we could add it as one key, the "~" acting as accent_tilde instead of adding it twice.
In my opinion this would make easy to visualize at keyboard and would bring the same behavior as we have at computer keyboards.